packages/dropdowns/package.json

```
@@ -23,7 +23,7 @@
   "dependencies": {
     "@floating-ui/react-dom": "^2.0.0",
     "@zendeskgarden/container-combobox": "^2.0.5",
-    "@zendeskgarden/container-menu": "^1.0.0",
+    "@zendeskgarden/container-menu": "1.0.1",
```

```suggestion
    "@zendeskgarden/container-menu": "^1.0.1",
```

As a rule, Garden container dependencies get hat notation.

---

Sorry, old habits. 🤦